client/src/feedbacker-components/dashboard-link/dashboard-link.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+// Helpers
+import classNames from 'classnames/bind'
+// Styles
+import styles from './dashboard-link.scss'
+
+const css = classNames.bind(styles)
+
+function url() {
+  return `//${window.location.origin.match(/([^.]*).(.*)/m)[2]}`
```

Comment:
add comment, what is this? name function to base domain or something, not whole url